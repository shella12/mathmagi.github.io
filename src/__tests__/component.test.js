import React from 'react';
import renderer from 'react-test-renderer';
import { Link, MemoryRouter } from 'react-router-dom';

describe('test suite for components', () => {
    it('renders correctly home component', () => {
        const tree = renderer
        .create(<MemoryRouter><Link page="http://localhost:3000/">Home</Link></MemoryRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('renders correctly calculator component', () => {
        const tree = renderer
        .create(<MemoryRouter><Link page="http://localhost:3000/calculator">Calculator</Link></MemoryRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('renders correctly quote component', () => {
        const tree = renderer
        .create(<MemoryRouter><Link page="http://localhost:3000/quote">Quote</Link></MemoryRouter>)
        .toJSON();
        expect(tree).toMatchSnapshot();
    });
})