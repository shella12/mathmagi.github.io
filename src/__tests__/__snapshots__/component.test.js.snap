// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test suite for components renders correctly calculator component 1`] = `
<a
  href="/"
  onClick={[Function]}
  page="http://localhost:3000/calculator"
>
  Calculator
</a>
`;

exports[`test suite for components renders correctly home component 1`] = `
<a
  href="/"
  onClick={[Function]}
  page="http://localhost:3000/"
>
  Home
</a>
`;

exports[`test suite for components renders correctly quote component 1`] = `
<a
  href="/"
  onClick={[Function]}
  page="http://localhost:3000/quote"
>
  Quote
</a>
`;
